Orange/widgets/visualize/tests/test_owboxplot.py

```diff
@@ -176,6 +176,20 @@ def test_empty_groups(self):
         self.send_signal(self.widget.Inputs.data, table)
         self.assertEqual(2, len(self.widget.boxes))
 
+    def test_sorting_disc_group_var(self):
+        """Test if subgroups are sorted by their size"""
+        table = Table("adult")
```

Comment:
can you use adult_sample? loading adult takes 0.5s to load on my computer, while loading adult_samples is instant.

Reply:
Fixed, there really was no reason to use such a large dataset for testing this.